crates/yakui-core/src/input/mod.rs

```diff
@@ -5,3 +5,6 @@ mod input_state;
 
 pub use self::button::*;
 pub use self::input_state::*;
+
+pub use keyboard_types;
```

Comment:
Should we expose this whole crate or just the parts we use?

Reply:
Probably the latter

Reply:
I've force-pushed a version that removes this line